lib/Entity/Module.php

```diff
@@ -221,6 +221,16 @@ class Module implements \JsonSerializable
      */
     public $settings = [];
 
+    /**
+     * @SWG\Property(
+     *     description="An array of additional module specific group properties",
+     *     type="array",
+     *     @SWG\Items(type="string")
+     * )
+     * @var \Xibo\Widget\Definition\GroupProperty[]
+     */
+    public $propertyGroups = [];
```

Comment:
@maurofmferrao are we insisting that the property groups are defined at the module level? Or will we have those also on the Module Template and combine that on the front end when rendering the form?

Reply:
Hmm yeah, I think it would be nice to have them all in the templates as well 🤔

Reply:
I think it comes down to the question of whether as a template developer, would you want to add your own tabs, or be limited to the ones provided by the module developer?

Reply:
That a good question. I see the possibility of a template having too many properties and have to group them. Something like Metro Static template can use different groups than other "Twitter" widgets, So maybe we should also had that property groups for the template as well